### ISSUE TITLE
Asking about the gan Training 

### ISSUE THREAD

**Issue by nehdiii** [ssundaram21/6.819FinalProjectRAMP#2]
Hello I'm trying to implement this method and re train the GAN so I'm facing an implementation problem 

I'm using ChestX-ray14 dataset its the same like chexpert so i prepossessed the data to have a vector of length 15 

![image](https://user-images.githubusercontent.com/74627775/183292083-248993ce-6fda-4b61-b5c3-706aefda2163.png)

this is simple shot about dataset i prepared 

now i build the generator network and discriminator network then i started the training loop 

![image](https://user-images.githubusercontent.com/74627775/183292195-a03d8bd5-5772-47a2-9b7c-e9fa145204ce.png)

this is the beginning of my training loop so i want to start with the generator update the problem is i need to input a random labels per class i one hot encode it concatenate with latent vector then pass it to the generator in the paper you mentioned that i need to pass the 15 dimension vector so my question is : 

How to pass random labels to the generator
- i create a vector with 15 dim and i randomly put 0 and 1 at each position 
or there is a particular methode a

and thank you
